sdcm/utils/verify.py

```diff
@@ -0,0 +1,30 @@
+from difflib import unified_diff
+from io import StringIO
+from pathlib import Path
+
+import click
+
+
+class Fix(StringIO):
```

Comment:
the name `Fix` is meaningless and not close to the actual thing this class is doing....